### DISCUSSION TITLE
Microsoft certification error report

### DISCUSSION THREAD

**Discussion by zhouzhiouhub** [microsoft/WindowsAppSDK#4961] [Q&A]
I keep getting errors when uploading a self-research project to the Microsoft Store. Is there anyone who has encountered the same problem? How did they solve it?

"Products may depend on non-integrated software (such as another product or module) to deliver primary functionality only as long as the additional required software is disclosed within the first two lines of the description in the Store.

Undisclosed software: Visual C++ Redistributable x86
Tested devices: Lenovo ThinkPad Yoga P40"
![image](https://github.com/user-attachments/assets/f97d43c0-8796-4cff-a2c7-bd9963eb6f6a)

**Comment by DrusTheAxe** [comment]
Are you making a packaged or unpackaged app?

If packaged you don't need the C++ runtime, you need your `appxmanifest.xml` declaring a dependency on the VCLibs package to satisfy your CRT needs.

If unpackaged then sorry, I'm not familiar with Store's current processes or policies re unpackaged app submission
